stubs/pyxdg/xdg/MenuEditor.pyi

```diff
@@ -0,0 +1,147 @@
+from os import PathLike
+from typing import overload
+from typing_extensions import TypeAlias
+
+from stdlib.xml.etree.ElementTree import ElementTree
```

Comment:
We do imports in the stdlib in typeshed the same as you'd do them at runtime :) Type checkers won't recognise a "stdlib" package

```suggestion
from xml.etree.ElementTree import ElementTree
```